### ISSUE TITLE
Occassional segfault with v0.24.0

### ISSUE THREAD

**Issue by thenonameguy** [F1bonacc1/process-compose#25]
Example stacktrace.
```
zerolog: could not write event: write /tmp/process-compose-kszabo.log: file already closed
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xa9f67c]

goroutine 1 [running]:
github.com/f1bonacc1/process-compose/src/tui.(*pcView).handleShutDown(0x0)
	github.com/f1bonacc1/process-compose/src/tui/view.go:132 +0x9c
github.com/f1bonacc1/process-compose/src/tui.Stop(...)
	github.com/f1bonacc1/process-compose/src/tui/view.go:389
github.com/f1bonacc1/process-compose/src/cmd.runTui(0xc00042bf40)
	github.com/f1bonacc1/process-compose/src/cmd/root.go:119 +0xda
github.com/f1bonacc1/process-compose/src/cmd.glob..func3(0x1a1c520?, {0xbea3a3?, 0x2?, 0x2?})
	github.com/f1bonacc1/process-compose/src/cmd/root.go:72 +0x387
github.com/spf13/cobra.(*Command).execute(0x1a1c520, {0xc000032190, 0x2, 0x2})
	github.com/spf13/cobra@v1.6.0/command.go:920 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0x1a1c520)
	github.com/spf13/cobra@v1.6.0/command.go:1040 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.6.0/command.go:968
github.com/f1bonacc1/process-compose/src/cmd.Execute({0xd13d28?, 0xc0000101a8?})
	github.com/f1bonacc1/process-compose/src/cmd/root.go:87 +0x4a
main.main()
	github.com/f1bonacc1/process-compose/src/main.go:37 +0x98

```

**Comment by F1bonacc1** [comment]
@thenonameguy sorry for that.
Fixed in v0.24.1

**Comment by thenonameguy** [comment]
Resolved as far as I can tell, thanks!
